Packs/InfoArmor_VigilanteATI/Integrations/InfoArmorVigilanteATI/InfoArmorVigilanteATI.py

```diff
@@ -0,0 +1,744 @@
+
+
+def get_leak_command(args):
+    leak_id = args.get('leak_id')
+    domains = args.get('domains')
+    token = args.get('token')
+    limit = int(args.get('limit', 20))
+
+    leak = get_leak(leak_id, limit, domains, token)
+    if not leak and len(leak.get('accounts')) == 0:
+        demisto.results('Leak has no accounts related to it')
+        sys.exit(0)
+
+    accounts = leak.get('accounts')
+    for i, _ in enumerate(accounts):
+        account = accounts[i]
+        account['email'] = account['plain']
+        del account['plain']
+
+    markdown = tableToMarkdown('Accounts related to leak {}'.format(
+        leak_id), leak.get('accounts'), ['email', 'domain', 'password', 'type_id'])
+    outputs = {
+        'VigilanteATI.Leaks(val.leak_id === obj.leak_id)': {
+            'leak_id': leak_id,
+            'accounts': accounts
+        }
+    }
+    if leak.get('token'):
+        outputs['VigilanteATI.LeakAccountsToken(true==true)'] = leak.get('token')
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': leak,
+        'HumanReadable': markdown,
+        'EntryContext': outputs
+    })
+
+
+def remove_none_params(params_dict):
+    """
+    filter only the params that have values
+    """
+    return dict((k, v) for k, v in params_dict.items() if v is not None)
+
+
+def query_ecrime_intelligence_database(query, q_forum, q_start_data, limit, re_token=None):
+    params = remove_none_params({
+        'query': query,
+        'q_forum': q_forum,
+        'q_start_data': q_start_data,
+        'limit': limit,
+        're_token': re_token
+    })
+
+    res = http_request('get', 'api/1/ecrime/posts/query', 'ecrime.posts.query', params)
+    return res.json()
+
+
+def query_ecrime_intelligence_database_command(args):
+    query = args.get('query')
+    q_forum = args.get('q_forum')
+    q_start_data = args.get('q_start_data')
+    limit = int(args.get('limit', 10))
+    re_token = args.get('re_token')
+
+    results = query_ecrime_intelligence_database(query, q_forum, q_start_data, limit, re_token)
+    posts = results.get('posts')
+    for post in posts:
+        post['title'] = base64.b64decode(post['title']).decode('utf8')
+        post['post'] = base64.b64decode(post['post']).decode('utf8')
+
+    markdown = tableToMarkdown('ECrime Posts', posts)
+    markdown += '\n**Total Count**: {}'.format(results.get('count'))
+    markdown += '\n**Next Page Token**: \n{}'.format(results.get('re_token'))
+
+    outputs = {
+        'VigilanteATI.ECrimePosts': posts
+    }
+    if results.get('re_token'):
+        outputs['VigilanteATI.ECrimeQueryToken(true==true)'] = results.get('re_token')
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': results,
+        'HumanReadable': markdown,
+        'EntryContext': outputs
+    })
+
+
+def list_account_credentials(leak_id, limit, domains, token=None):
```

Comment:
This method seems to be identical to `get_leak`

Reply:
Deleted the 'list_account_credentials' method because it is not used.